### DISCUSSION TITLE
Option to set a deployment/application as a non-web application

### DISCUSSION THREAD

**Discussion by OG-Jons** [coollabsio/coolify#2467] [Feature Requests]
Hi there!
It would be great to have a toggle switch, that allows users to toggle an application being web-accessible or not (e.g if it's a bot that is running it obviously doesn't have a webui)
Right now when nothing is configured, coolify complains (and rightly so) that the app is in an unhealthy state, because it can't reach an endpoint for the healthcheck.
I remember coolify v3 having this, so it would be nice to have this in a shape or form again.

**Comment by andrasbacsai** [comment]
Hey, you could have that in v4, by not setting a domain. 

The healthcheck is using internal connection, so it could check your application inside the container, even there is no domain set for it.
